official/recommendation/ncf_common.py

```diff
@@ -312,6 +313,11 @@ def eval_size_check(eval_batch_size):
   def xla_validator(flag_dict):
     return not flag_dict["use_xla_for_gpu"] or not flag_dict["tpu"]
 
+  flags.DEFINE_bool(
+      name="cloning", default=True, help=flags_core.help_wrap(
```

Comment:
can we call it something more explicit like "clone_model_in_keras_dist_strat"?

Reply:
Actually we should/need to move this option into the "global" flags because it would/could also be used with transformer and is already used for NCF and ResNet.  You just need to add it to the flag options under /utils and then each model indicates it is a valid flag for their situation.  I can help tomorrow, this is a late night drive-by.  

To fix this correctly is bit of work that would be appreciated.  You need to move this [flag logic](https://github.com/tensorflow/models/blob/master/official/resnet/resnet_run_loop.py#L709)  out of run_loop and into something like "resnet_common" so it is not so tightly associated with the estimator version and have it include everything that is mostly common for the resnet models (keras and estimator).  The other option is to just copy everything you need for keras into keras_common.define_flags.  I would be ok with the `common_resnet` was in the keras folder.  

I am all for a more descriptive name, I agree now that cloning is to vague.  Priya's name is also a bit long, maybe we can ignore tf 1.0 and call it dist_strat_clone_model. 

Sorry this got long, I tried to clean up my wording.  It is worth it to take 2 hours or so can clean this up before it gets more messy.

Reply:
I renamed the flag.

We clarified offline the temporary nature of the flag and decide to leave the current flag plumbing as is.